src/backend/web/templates/bracket_partials/double_elim_bracket_table.html

```diff
@@ -85,13 +104,7 @@
       <td colspan="4"></td>
       <td class="dash" colspan="2"></td>
       <td rowspan="2" class="match">
-        {% if event.year >= 2023 %}
-        <span class="match-label">Match 11</span>
-        {{ bracket_match(bracket_table.sf.get('sf11'), 'Winner of M10', 'Winner of M9') }}
-        {% else %}
-        <span class="match-label">Match 12</span>
-        {{ bracket_match(bracket_table.sf.get('sf12'), 'Winner of M7', 'Winner of M8') }}
-        {% endif %}
+        {{ bracket_match('Match 11', bracket_table.sf.get('sf11'), 'Winner of M7', 'Winner of M8') }}
```

Comment:
Deleted the `if` since there is a separate bracket for pre 2023:

https://github.com/the-blue-alliance/the-blue-alliance/blob/dc78856e088f1caa09e3fa1a7fd5addbe80c8f65/src/backend/web/templates/bracket_partials/double_elim_bracket_table_pre_2023.html